manifests/nightly/postgresql-operator.v0.0.10.clusterserviceversion.yaml

```diff
@@ -1,23 +1,23 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
-  name: REPLACE_NAME.vREPLACE_VERSION-REPLACE_TAG
```

Comment:
Please keep the placeholders as they are properly replaced by the release process. thx.

Reply:
I've reverted those placeholders in https://github.com/Avni-Sharma/postgresql-operator/pull/1